app/src/main/res/values/strings.xml

```diff
@@ -188,22 +188,22 @@
     <string name="bsct_conn_rule_explanation">
         &lt;u>#1&lt;/u>: Blocks an app from connecting to the Internet.
         \n &lt;u>#2&lt;/u>: Blocks any app from connecting to a particular IP.
-        \n &lt;u>#3&lt;/u>: Blocks all connections when device locked (Universal Firewall).
-        \n &lt;u>#4&lt;/u>: Blocks any app not in use (Universal Firewall).
-        \n &lt;u>#5&lt;/u>: Blocks connections when source app unknown (Universal Firewall).
-        \n &lt;u>#6&lt;/u>: Blocks all UDP traffic except DNS (Universal Firewall).
-        \n &lt;u>#7&lt;/u>: Block connections when DNS is bypassed (Universal Firewall).
+        \n &lt;u>#3&lt;/u>: Blocks all connections when device locked (Universal firewall).
+        \n &lt;u>#4&lt;/u>: Blocks any app not in use (Universal firewall).
+        \n &lt;u>#5&lt;/u>: Blocks connections when source app unknown (Universal firewall).
+        \n &lt;u>#6&lt;/u>: Blocks all UDP traffic except DNS (Universal firewall).
+        \n &lt;u>#7&lt;/u>: Block connections when DNS is bypassed (Universal firewall).
         </string>
 
-    <string name="private_dns_toast">Private DNS has been set.</string>
-    <string name="always_on_dialog_heading">Disable Always-on VPN</string>
-    <string name="always_on_dialog">\"Always-On VPN\" is enabled on this device. To start Rethink, please go to Settings app and disable \"Always-on VPN\".</string>
+    <string name="private_dns_toast">Private DNS set.</string>
+    <string name="always_on_dialog_heading">Action needed</string>
+    <string name="always_on_dialog">\"Always-On VPN\" is turned on for this device. To start Rethink, please go to the Android Settings and turn off \"Always-on VPN\".</string>
     <string name="always_on_dialog_positive_btn">Go to Settings</string>
     <string name="always_on_dialog_negative_btn">Cancel</string>
 
     <string name="always_on_dialog_stop_heading">Always-on VPN</string>
-    <string name="always_on_dialog_stop_message">Always-on VPN is enabled for Rethink. Are you sure you want to stop Rethink anyway?</string>
-    <string name="always_on_dialog_lockdown_stop_message">RethinkDNS is running VPN in <i>Lockdown</i> mode. Stopping it would result in loss of Internet connectivity on this user profile. Do you want to continue anyway?</string>
+    <string name="always_on_dialog_stop_message">Stop Rethink even though it uses always-on VPN in Android?</string>
```

Comment:
Not sure about this.

Reply:
How is it then "enabled for Rethink"?
I don't understand the implications here.